CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 + components: Add `icon` to `IconButton`
++ core: Support adding children to libpanel's widgets 
```

Comment:
This is a bit misleading. You could already add children to those widgets before, but now the extension traits are in place to make it more convenient. I would rather write something like "implement container traits to allow easier usage of libpanel widgets in the view macro".